coral-component-list/src/scripts/SelectList.js

```diff
@@ -218,13 +218,41 @@ class SelectList extends BaseComponent(HTMLElement) {
   get _tabTarget() {
     return this.__tabTarget || null;
   }
+  
   set _tabTarget(value) {
     this.__tabTarget = value;
-  
-    // Set all but the current set _tabTarget to not be a tab target:
-    this.items.getAll().forEach((item) => {
-      item.setAttribute('tabindex', item === value ? 0 : -1);
-    });
+    if (this.groups && this._groups.getAll().length > 0) {
```

Comment:
I think this change to set focusable items inside a group should go inside function _resetTabTarget.

function _tabTarget  has an argument 'value' which should be set as tabTarget or if this isn't inside the list then we should call _resetTabTarget.